sim/midas/src/main/scala/midas/platform/PlatformShim.scala

```diff
@@ -43,4 +45,9 @@ abstract class PlatformShim(implicit p: Parameters) extends LazyModule()(p) {
 
     top.module.headerConsts map vMacro foreach sb.append
   }
+
+  // Emit a `XDCPathToCircuitAnnotation` with the pre- and post-link circuit
+  // paths provided in the configuration. These locate the design within the
+  // context of the larger FPGA design.
+  SpecifyXDCCircuitPaths(p(PreLinkCircuitPath), p(PostLinkCircuitPath))
```

Comment:
You might as well remove `SpecifyXDCCircuitPaths` too right? Or better yet, add a deprecation warning to it.

Reply:
The `WriteXDCFile` pass still relies on that annotation to determine the paths. I can switch that over to just use the parameters directly if we want to entirely deprecate that `XDCPathToCircuitAnnotation`!

Reply:
Ah yeah, right. Nvm me. Lets leave it for now.